src/Login.ts

```diff
@@ -179,26 +185,30 @@ export default class Login {
     }
 }
 
+/**
+ * Describes the OIDC native login flow
+ * Separate from js-sdk's `LoginFlow` as this does not use the same /login flow
+ * to which that type belongs.
+ */
 export interface OidcNativeFlow extends ILoginFlow {
     type: "oidcNativeFlow";
+    // this client's id as registered with the configured OIDC OP
     clientId: string;
 }
 /**
  * Finds static clientId for configured issuer, or attempts dynamic registration with the OP
+ * Returns OIDC native flow when client is ready to attempt login via OIDC native flow
```

Comment:
I'd find this easier to grok the other way around. To my mind, the primary purpose of the function is to return the `OidcNativeFlow`. Sure, it also finds the clientId or registers, but that's just one step along the way.

```suggestion
 * Prepares an OidcNativeFlow for logging into the server.
 * 
 * Finds a static clientId for configured issuer, or attempts dynamic registration with the OP, and wraps the
 * results.
```